x-pack/plugins/enterprise_search/public/applications/shared/error_state/error_state_prompt.tsx

```diff
@@ -38,58 +44,22 @@ export const ErrorStatePrompt: React.FC = () => {
           <p>
             <FormattedMessage
               id="xpack.enterpriseSearch.errorConnectingState.description1"
-              defaultMessage="We can’t establish a connection to Enterprise Search at the host URL: {enterpriseSearchUrl}"
+              defaultMessage="We can’t establish a connection to Enterprise Search at the host URL {enterpriseSearchUrl} due to the following error."
               values={{
-                enterpriseSearchUrl: <EuiCode>{config.host}</EuiCode>,
+                enterpriseSearchUrl: (
+                  <EuiLink
+                    target="_blank"
+                    href={config.host || ''}
+                    css={{ overflowWrap: 'break-word' }}
+                  >
+                    {config.host}
+                  </EuiLink>
```

Comment:
If there is a fallback to the `config.host` property, shouldn't we not show the link at all if it is not present?

Reply:
`config.host` will always be present when we're showing the error message. We probably don't actually need this check here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search/index.tsx#L33